light/odr.go

```diff
@@ -28,9 +28,9 @@ import (
 	"github.com/ethereum/go-ethereum/ethdb"
 )
 
-// NoOdr is the default context passed to an ODR capable function when the ODR
-// service is not required.
-var NoOdr = context.Background()
+// DefaultContext is the default context passed to an ODR capable function when
+// the ODR service has no latency limitation.
+var DefaultContext = context.Background()
```

Comment:
Now that the default context has no special meaning (which is good) maybe we should drop this definition entirely.